simpleflow/history.py

```diff
@@ -155,6 +155,8 @@ def get_activity():
                 "decision_task_completed_event_id": event.decision_task_completed_event_id,
             }
             if event.activity_id not in self._activities:
+                activity["retry"] = 0
+                activity["attempt"] = 1
```

Comment:
Maybe we could use the `attempt_number` name, as in `simpleflow.task.TaskFailureContext`?
And update this class not to use `retry`? 🙂